### PR TITLE
Add person student  teacher informatin

### PR DESCRIPTION
**Pull request by faisalakb**
In this project apply the following features: :100: 

Create class Person with the following:

    Instance vars: @id, @name, and @age.
    Constructor with name, age, and parent_permission as parameter. name and parent_permission are optional and have default values of "Unknown" and true.
    Getters for @id, @name, and @age.
    Setters for @name and @age.
    Private method of_age? that returns true if @age is greater or equal to 18 and false otherwise.
    Public method can_use_services? that returns true if person is of age or if they have permission from parents.

Create class Student with the following:

    Inherits from Person.
    Constructor extends parent's constructor by adding @classroom and a parameter for it.
    Method play_hooky that returns "¯\(ツ)/¯".

Create class Teacher with the following:

    Inherits from Person.
    Constructor extends parent's constructor by adding @specialization and a parameter for it.
    Override can_use_services? so it always returns true.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,117 @@
...
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+- I would like to thank Microverse and  all code reviewers for making this project better.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📝 License <a name="license"></a>
+
+
+This project is [MIT](https://github.com/faisalakb/Ruby_OOP_school_library/blob/dev/LICENSE) licensed.
```
Comment (shahadat3669) [inline README.md:115]: - [ ] 👏 Great job on including a custom license file for your project. However, it seems that the link to the license file is broken in the README. To make your project more professional and ensure that users can access the license, please provide a working link to the license file in the README's license section. This is an important aspect of projects and will help users understand how they can use your code. Keep up the good work! 👍📜👩‍💻

**Review by shahadat3669 — CHANGES_REQUESTED** [review]
Hi @faisalakb , 

Great job so far! 👏 You've done well, and you're almost there for the next project ⏭️, but there are a few issues 🐛 that you still need to work on.




## Don't Worry 🙏

It's okay to have bugs in your code. 🆗 Please see this code review as an opportunity to learn practical coding skills and discover new tips that can improve your code quality 💯 and help you **get even better** 💼.

**I will guide you on how to address these areas down below 👇.**

## Highlights

✅ There is an individual .rb file for each class (Person, Student, and Teacher )
✅ Created `person`, `student`, and `teacher` classes with necessary data.
✅ Public and Private methods are correctly defined.


## Required Changes ♻️

- [ ] 👍 Excellent work on your project setup, particularly the linters workflow file. You've done a great job! However, to ensure a comprehensive project setup that aligns with the [project requirements](https://github.com/microverseinc/linters-config/blob/master/ruby/.github/workflows/tests.yml), kindly consider adding the `linters.yml` file to the GitHub workflows folder. I know this `test.yml` file is not useful in this milestone. But in a later milestone, you'll be testing all your classes, so including this file is essential. Keep up the fantastic work! 👏🔧🚀

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

## Have any doubt ❓

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

<table width="100%" align="center">
  <tr>
    <td colspan="4" align="center">
      <strong>Contact me: Shahadat Hossain</strong>
    </td>
  </tr>
  <tr>
    <td align="center">
      <a href="https://github.com/shahadata3669">
        GitHub
      </a>
    </td>
    <td align="center">
      <a href="https://www.linkedin.com/in/shahadat-cseng/">
        LinkedIn
      </a>
    </td>
    <td align="center">
      <a href="https://app.slack.com/client/T47CT8XPG/D05GT4E4WAY/rimeto_profile/U04426SD71V">
        Slack
      </a>
    </td>
    <td align="center">
      <a href="mailto:shahadat3669@gmail.com?subject=Microverse%20Code%20Review">
        Email
      </a>
    </td>
  </tr>
</table>

## ⚠️ WARNING ⚠️

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by zeff96 — APPROVED** [review]
### Status: APPROVED :green_circle: 
Hi @faisalakb , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

congrats

## Highlights
✔️ Linters passing
✔️ Correct workflow
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by faisalakb** [comment]
Thanks for the time

**Pull request merged**